src/Simulation/QsharpCore/Intrinsic.qs

```diff
@@ -118,10 +16,7 @@ namespace Microsoft.Quantum.Intrinsic {
     /// This is a no-op. It is provided for completeness and because
     /// sometimes it is useful to call the identity in an algorithm or to pass it as a parameter.
     operation I (target : Qubit) : Unit
-    is Adj + Ctl {
-        body (...) { }        
-        adjoint self;
-    }
+    is Adj + Ctl { }
```

Comment:
We actually will want to keep the explicitly "adjoint self" - it is the only way currently to declare that an operation is self-adjoint (which is information that can be used for optimization and that the compiler propagates).

Reply:
Ah, I didn't know that. I will put this back.